pyproject.toml

```diff
@@ -6,7 +6,6 @@ requires = [
 
 [project]
 name = "pyproject-fmt-rust"
-version = "1.0.1"
```

Comment:
Do you have a link where they say you can do that? 🤣 At the very least the version should be dynamic otherwise it's violating pep 621...

Reply:
It is. Check below. ;) It previously was violating PEP 621 by having both static and dynamic versions.

https://www.maturin.rs/metadata (at the top)

Reply:
I can't comment on it, but see line 32.